posthog/tasks/exports/csv_exporter.py

```diff
@@ -210,7 +211,7 @@ def make_api_call(
 ) -> requests.models.Response:
     request_url: str = absolute_uri(next_url or path)
     try:
-        url = add_query_params(request_url, {"limit": str(limit)})
+        url = add_query_params(request_url, {"limit": str(limit), "is_csv_export": "1"})
```

Comment:
This is nice. We can use this pattern elsewhere to tidy up some of the formatting